topic-operator/src/main/java/io/strimzi/operator/topic/ZkTopicsWatcher.java

```diff
@@ -146,13 +146,17 @@ public void handle(AsyncResult<List<String>> childResult) {
                     tcw.addChild(topicName);
                     tw.addChild(topicName);
                     LogContext logContext = LogContext.zkWatch(TOPICS_ZNODE, watchCount + ":+" + topicName);
-                    topicOperator.onTopicCreated(logContext, new TopicName(topicName)).onComplete(ar -> {
-                        if (ar.succeeded()) {
-                            LOGGER.debug("{}: Success responding to creation of topic {}", logContext, topicName);
-                        } else {
-                            LOGGER.warn("{}: Error responding to creation of topic {}", logContext, topicName, ar.cause());
-                        }
-                    });
+                    if ("__consumer_offsets".equals(topicName) || "__transaction_state".equals(topicName)) {
+                        LOGGER.debug("{}: Cannot create internal topic {}", logContext, topicName);
```

Comment:
It's not enough to log this (even at INFO or WARN), because the user manipulating the `KafkaTopic` might not have access to the TO logs. We need to something in `status.conditions`. 

It's also not enough to just have this in the `ZkTopicsWatcher`. You can't eliminate the possibility that the topic creation is handled first by the periodic reconciliation. Probably something in `TopicOperator.reconcile` will allow you to make the check in a single place.

Reply:
This exception is thrown in the cluster where no custom `KafkaTopic` resources have been created. What happens is

- kafka creates internal topic `__consumer_offsets`
- TO watches this and creates k8s resource + entry in the ZK
- during that, `UnknownTopicOrPartitionException` is thrown